src/Modules/Guilds/Hooks/types.ts

```diff
@@ -0,0 +1,32 @@
+export type WagmiUseContractReadResponse<T> = Pick<
+  import('react-query').QueryObserverResult<

```

Comment:
I would have preferred if you only extracted the imported values instead of using them directly like this (for clarity). But that's just a personal opinion :).

Reply:
Mmm. I know is messy, but the best way to pull the real values wagmi is using. 
I will use a custom `WagmiUseContractReadResponse` with a TODO to use wagmi type when ready and remove our internal type. 
```
export type WagmiUseContractReadResponse<T> = {
  data: T;
  error: any;
  isError: boolean;
  isFetched: boolean;
  isFetching: boolean;
  isLoading: boolean;
  isRefetching: boolean;
  isSuccess: boolean;
  refetch: (options?: any) => Promise<any>;
  status: 'error' | 'success' | 'idle' | 'loading';
  fetchStatus: 'fetching' | 'paused' | 'idle';
};
```